climpred/prediction.py

```diff
@@ -223,13 +222,15 @@ def compute_hindcast(
 
     # take only inits for which we have verification data at all leads
     if alignment == 'same_inits':
-        forecast, verif = reduce_time_series_for_aligned_inits(forecast, verif, nlags)
+        forecast, verif = reduce_forecast_to_same_inits(forecast, verif)
+    else:
+        raise NotImplementedError(
+            "Only 'same_inits' alignment is currently implemented."
+        )
 
     plag = []
```

Comment:
maybe we can add a first logging message before informing about `mse.name`, `comparison.name`, `alignment` and the like. When logging and running to skill computations right after another, a clear separation would be useful

Reply:
I actually had this here and took it away for some reason. I agree completely. I'll add it back in. Because if you save out to a log it just keeps overwriting that log with new inits and it's hard to know what it's from.